duckduckgo/privacy-dashboard
# Phishing Warning

**Reviewer:** @shakyShane 

https://app.asana.com/0/1206594217596623/1207336102226042/f

## Description:

Phishing warning state for Privacy Dashboard

## Steps to test this PR:

1. Run `npm run preview`
2. Select macOS
3. Select `phishing` from the dropdown
4. Confirm that phishing warning is shown
